specs/v1.yaml

```diff
@@ -433,6 +481,54 @@ paths:
           $ref: "#/components/responses/NotFoundError"
         "500":
           $ref: "#/components/responses/InternalServerError"
+    put:
+      summary: Update Unintegrated Grant
+      description: |-
+        Update an unintegrated grant object generated by Chariot Connect.
+        This is useful to update the status or acknowledgement of the unintegrated grant.
+        If the unintegrated grant does not exist, returns a 404 status.
+        If the provided ID is not a v4 UUID according to RFC 4122, returns a 400 status.
+      operationId: update-unintegrated-grant
+      tags:
+        - Grants
+      security:
+        - auth0:
+            - "update:grants"
+          api_key: []
+      parameters:
+        - name: id
+          in: path
+          description: the unique id of the unintegrated grant
+          schema:
+            type: string
+            format: uuid
+          required: true
+          example: 10229488-08d2-4629-b70c-a2f4f4d25344
+      requestBody:
+        $ref: "#/components/requestBodies/UpdateGrantRequest"
```

Comment:
is this correct or should there be an body for unintegrated grants

Reply:
the body is the same between Grants and Unintegrated Grants for update (this is assuming we're adding statuses to Unintegrated Grants which we will be doing..)